.github/workflows/test.yaml

```diff
@@ -17,7 +17,7 @@ jobs:
 
       - name: Run tests
         run: |
-          spread google:
+          spread google:ubuntu-20.04-64: google:ubuntu-22.04-64-devstack:tests/openstack
```

Comment:
Hi, thanks for the review, 

In order to test openstack backend against a real openstack interface, we pre-configured a new image with devstack already installed. The main raeson was to speed up and simplify the test (so that image is only used to test openstack). Should I add this explanation in the workflow?